contracts/root/withdrawManager/WithdrawManager.sol

```diff
@@ -233,11 +233,9 @@ contract WithdrawManager is WithdrawManagerStorage, IWithdrawManager {
             exitQueue.delMin();
             // If the exitNft was deleted as a result of a challenge, skip processing this exit
             if (!exitNft.exists(exitId)) continue;
-
+            address exitor = exitNft.ownerOf(exitId);
+            exits[exitId].owner = exitor;
```

Comment:
Do we need to change owner in exists?  @atvanguard @AyushKaul

Reply:
its used while encoding so changed it.
cc: @atvanguard

Reply:
Do we need it here though?